src/test/java/edu/wisc/my/restproxy/service/RestProxyServiceImplTest.java

```diff
@@ -60,12 +63,33 @@ public void proxyRequest_control() {
   }
   
   /**
+   * Test simulates a proxy request which fails with a http 400 error.  
+   * An error like this could be encountered if you were to post invalid data to a form.
+   * Test verifies the HttpStatus AND the body (which likely contins validation error message) are passed back to the client.
+   */
+  @Test
```

Comment:
I agree the tests are a bit weak, since they are just doing a `verify(proxyDao).proxyRequest(any(RequestContext.class))`. To continue unit testing, we'll need equals/hashCode on RequestContext so we can confirm some expected instances are getting past. Perhaps the RequestContext construction should be isolated and tested separately.

I'll second the need for better testing - see issue #18.

This gets a :+1: from me, I'll merge and release it.